### PR TITLE
[VCRUNTIME] Add more definitions

### PR DESCRIPTION
**Pull request by tkreuzer**
## Purpose

Add more definitions to vcruntime headers. This is to be able to build the UCRT code.

### CODE REVIEW

File: sdk/include/vcruntime/eh.h [hunk 1]
```diff
@@ -24,10 +28,10 @@ typedef void (__cdecl *unexpected_handler)(void);
 struct _EXCEPTION_POINTERS;
 typedef void (__cdecl *_se_translator_function)(unsigned int,struct _EXCEPTION_POINTERS *);
 
-_CRTIMP __declspec(noreturn) void __cdecl terminate(void);
+_CRTIMP __declspec(noreturn) void __cdecl terminate(void) throw();
```
Comment (whindsaks) [inline sdk/include/vcruntime/eh.h:31]: std::terminate is noexcept in C++11, is there any chance these should be as well (I realize throw() is basically the same thing).

**Review by DarkFire01 — APPROVED** [review]

**Comment by DarkFire01** [comment]
Looks like for clang 32bit we need to up the target architecture feature level.

File: sdk/include/vcruntime/mmintrin.h [hunk 2]
```diff
@@ -59,9 +59,9 @@ extern "C" {
     typedef float __m64 __attribute__((__vector_size__(8), __aligned__(16)));
 
 #ifdef __clang__
-#define __INTRIN_INLINE_MMX __INTRIN_INLINE __attribute__((__target__("mmx"),__min_vector_width__(64)))
+#define __INTRIN_INLINE_MMX __INTRIN_INLINE __attribute__((__target__("mmx,sse2"),__min_vector_width__(64)))
```
Comment (learn-more) [inline sdk/include/vcruntime/mmintrin.h]: This seems a bit weird?

**Review by learn-more — APPROVED** [review]
Apart from one small things, this all seems like it makes sense.

**Review by HBelusca — APPROVED** [review]

**Pull request merged**